lib/ukrand/Config.uk

```diff
@@ -0,0 +1,56 @@
+menuconfig LIKUKRAND
+	bool "ukrand: Software random number generator"
+	default n
+
+if LIKUKRAND
+
+config LIBUKSWRAND
+	bool "Software-generated randomness"
+	default n
+
+if LIBUKSWRAND
+
```

Comment:
You can add an option for "Processor-generated randomness" and a help option that states that this is used pass-through.